client/src/redux/index.js

```diff
@@ -53,12 +55,14 @@ const initialState = {
 export const types = createTypes(
   [
     'appMount',
+    'hardGoTo',
+    'canRequestDonation',
```

Comment:
```suggestion
    'allowDonationRequests',
```
I named this poorly.